cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/sql/GcpDatabaseServerUpdateService.java

```diff
@@ -0,0 +1,49 @@
+package com.sequenceiq.cloudbreak.cloud.gcp.sql;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.google.api.services.sqladmin.SQLAdmin;
+import com.google.api.services.sqladmin.model.Operation;
+import com.google.api.services.sqladmin.model.User;
+import com.sequenceiq.cloudbreak.cloud.context.AuthenticatedContext;
+import com.sequenceiq.cloudbreak.cloud.exception.CloudConnectorException;
+import com.sequenceiq.cloudbreak.cloud.gcp.client.GcpSQLAdminFactory;
+import com.sequenceiq.cloudbreak.cloud.gcp.util.GcpStackUtil;
+import com.sequenceiq.cloudbreak.cloud.model.DatabaseServer;
+import com.sequenceiq.cloudbreak.cloud.model.DatabaseStack;
+
+@Component
+public class GcpDatabaseServerUpdateService extends GcpDatabaseServerBaseService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GcpDatabaseServerUpdateService.class);
+
+    @Inject
+    private GcpSQLAdminFactory gcpSQLAdminFactory;
+
+    @Inject
+    private GcpStackUtil gcpStackUtil;
+
+    public void updateRootUserPassword(AuthenticatedContext ac, DatabaseStack databaseStack, String newPassword) {
+        SQLAdmin sqlAdmin = gcpSQLAdminFactory.buildSQLAdmin(ac.getCloudCredential(), ac.getCloudCredential().getName());
+        String projectId = gcpStackUtil.getProjectId(ac.getCloudCredential());
+        DatabaseServer databaseServer = databaseStack.getDatabaseServer();
+        String instance = databaseServer.getServerId();
```

Comment:
is there a reason for putting these outside of try block?

Reply:
I just followed the already existing style and put only the provider calls in the try-catch block.

Reply:
discussed, makes sense